scripts/ethereum/deployment/src/deploy_contracts.ts

```diff
@@ -53,18 +57,20 @@ export async function main(args: ReadonlyArray<string>): Promise<void> {
     console.log(`${name} deployed to`, debugAddress(receipt.contractAddress));
 
     if (typeof contract.methods.mint !== "undefined") {
-      const mintingJobs: ReadonlyArray<MintingJob> = [
-        { recipient: secondIdentity, quantity: "33445566" },
-        { recipient: Address.fromString("0x0000000000111111111122222222223333333333"), quantity: "38" },
-      ];
-
-      for (const { recipient, quantity } of mintingJobs) {
-        console.log(`Minting ${quantity} atomic units for ${recipient} ...`);
-        await contract.methods
-          .mint(recipient, quantity)
-          .send({ from: mainIdentity, gasPrice: ganacheGasPrice })
-          .getReceipt();
-      }
+      mintingJobs.push(
+        { contract: contract, recipient: mainIdentity, quantity: "100000000" /* 100 million atomics */ },
+        { contract: contract, recipient: secondIdentity, quantity: "33445566" },
+        { contract: contract, recipient: noEthAddress, quantity: "38" },
+      );
     }
   }
+
+  // Perform all minting jobs after all deployments to not change delployment nonces when updating minting
```

Comment:
"Perform minting jobs after all deployments so that deployment nonces stay constant"?